src/QsFmt/Formatter/SyntaxTree/Namespace.fs

```diff
@@ -12,9 +12,11 @@ type TypeParameterBinding =
         CloseBracket: Terminal
     }
 
+type internal SpecializationParameter = Terminal
```

Comment:
I'm not sure I see the need for this wrapper type. If we keep the `Provided` case of type `Terminal Tuple option * Statement Block` would that break anything?

Reply:
No, everything will be fine. Actually, I just used `Terminal` for rewriting and reducing. I have no problem with removing it.

Reply:
Thanks!